### DISCUSSION TITLE
Duplicate IP Addresses between multiple Tenants

### DISCUSSION THREAD

**Discussion by mmanzano-dp** [netbox-community/netbox#11792] [Q&A] [answered]
Hello everyone, 

I'm new to the netbox world; sorry if this is not the best way to ask this question. 

We have multiple sites where the Prefixes and IPs for the network equipment are the same; since they are private, there is no conflict with other networks. 

I've created two separate Tenenat and Sites for each property on netbox, and creating two prefixes

![image](https://user-images.githubusercontent.com/125931207/220211806-0ba8bb70-65e3-4c01-94e0-c3f7ac55f715.png)

But when configuring the IP Addresses for the devices, I can still see the other Tenant Prefixes and the Duplicate IP Addresses that are configured at the other site. 

![image](https://user-images.githubusercontent.com/125931207/220211874-736d038b-6cf3-41f9-b340-a63fc0d7ddbb.png)

Is this the expected behavior? or do I'm configuring something wrong?

**Comment by jcollie** [comment, marked as answer]
I believe that you need to put each prefix in a separate VRF. Since the prefixes and IPs are in the same "Global" VRF NetBox believes that they are mutually reachable.

**Reply by mmanzano-dp** [reply]
Thanks for your help, this resolved the issue
